src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1541,35 +1541,52 @@ def create_function(cmd, resource_group_name, name, storage_account, plan=None,
             raise CLIError("Location is invalid. Use: az functionapp list-consumption-locations")
         functionapp_def.location = consumption_plan_location
         functionapp_def.kind = 'functionapp'
+        if is_linux and not runtime:
+            raise CLIError("usage error: --runtime required. Supported values 'dontnet', 'javascript', 'python'.")
```

Comment:
`node`, instead of javascript

Reply:
did we change this again? Portal is using javascript as well.

Reply:
i see in runtime PR that this node will update this & notify portal folks as well

Reply:
I thought @agruning and @hartra344  was aware of that to. But yes, values are `dotnet`, `node`, `python`